src/lib/Components/Inbox/Conversations/__tests__/Composer-tests.tsx

```diff
@@ -24,4 +26,13 @@ describe("regarding the send button", () => {
 
     expect(tree).toMatchSnapshot()
   })
+
+  it("calls onSubmit with the text when send button is pressed", () => {
+    const onSubmit = jest.fn()
```

Comment:
This tests the Composer `onSubmit` event handler gets called and behaves as intended when the send button is pressed